ios/ios/src/Storage/CMStore.m

```diff
@@ -631,13 +631,21 @@ - (void)saveUserObject:(CMObject *)theObject additionalOptions:(CMStoreOptions *
         }
         return;
     }
+
+    if (nil == theObject) {
+        return;
```

Comment:
What do you think about logging a debug message in this case? Often a nil object is indicative of a CM response that was unexpected (perhaps a malformed search query, etc).